pkg/debug/Dockerfile

```
@@ -38,8 +38,9 @@ COPY lshw/ lshw-B.${LSHW_VERSION}/
 WORKDIR /lshw-B.${LSHW_VERSION}
 
 # order is important
-RUN for patch in fix-musl-sc_long_bit.patch wrapper-for-basename.patch 15565229509455527de9ce7cbb9530e2b31d043b.patch\
- 2b1c730b493d647bbab4854713571458e82a81e7.patch; do patch -p1 < $patch; done &&\
+RUN for patch in *.patch; do \
+   patch -p1 < "$patch"; \
+ done &&\
```

```suggestion
 done && \
```
or even better:
```suggestion
  done
RUN make ...
```
(as I don't see why patch and make have to be started in the same RUN)

---

That's a yetus rule, we need to pack all commands into a single RUN....